src/inputParam/inputParamRoutines.F90

```diff
@@ -4042,6 +4042,9 @@ subroutine setDefaultValues
     usematrixfreedrdw = .False.
     sepSensorOffset = zero
     sepSensorSharpness = 10_realType
+    cavSensorOffset = zero
+    cavSensorSharpness = 10_realType
+    cavExponent = 0
```

Comment:
I'm pretty sure these lines are not necessary because the options get set in Python anyway

Reply:
Is it better to not have them? I followed the format for `sepSensorOffset` etc

Reply:
I think it's better to remove them. Right now, there are "default" values in three different places in the code and leaving them in there makes it unclear which ones actually do anything.